docs/coil.md

```diff
@@ -0,0 +1,154 @@
+# Coil for Jetpack Compose
+
+[![Maven Central](https://img.shields.io/maven-central/v/com.google.accompanist/accompanist-coil)](https://search.maven.org/search?q=g:com.google.accompanist)
+
+This library brings easy-to-use [Painter][painter] which can fetch and display images from external sources, such as network, using the [Coil][coil] image loading library.
+
+<img src="https://coil-kt.github.io/coil/logo.svg" width="480" alt="Coil logo">
+
+!!! info
+    If you're migrating from Accompanist 0.7.x or before, please read the [migration](./migration-coilimage) documentation after reading this document.
+
+## `rememberCoilPainter()`
+
+The primary API is via the [`rememberCoilPainter()`][rememberpainter] function. The simplest usage is like so:
+
+```kotlin 
+import androidx.compose.foundation.Image
+import com.google.accompanist.coil.rememberCoilPainter
+
+Image(
+    painter = rememberCoilPainter("https://picsum.photos/300/300"),
+    contentDescription = "My content description",
```

Comment:
What about using `stringResource` to promote best practice? Worth it?

Reply:
Done